docs/sdk/sdk-by-examples/simple-governance/testnet.md

```diff
@@ -0,0 +1,3 @@
+## Testnet
```

Comment:
just delete this if it's not ready

Reply:
But we'll get the question, so I feel it's good to have it apparent that it's WIP

Reply:
what do you mean?

Reply:
We will get questions around testet, how to create one, etc. If this is clear that this is WIP, we won't have to say it is WIP to everyone.

Reply:
but why is this the place to talk about testnets? it's in the SDK by examples section under simple governance?

and there is no content yet...

Reply:
ok now i have a better understand of where you're coming from. thank you. it would seem to me that at the end of "building a dApp" you might want to do any number of things with it. one of which is launching your own testnet. maybe you want to connect to the hub or connect to an existing testnet. we should have content for all of these options - but since we don't - i'd prefer to include a "TODO" in a production site. 

when will the TODO be done?

can we either wait until then or remove it for now?

Reply:
ok, I'll remove it for now

Reply:
thank you.